test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/liabilities/LiabilityDetailsViewSpec.scala

```diff
@@ -347,4 +347,44 @@ class LiabilityDetailsViewSpec extends ViewSpec with MockitoSugar {
 
   }
 
+  "Render view PDF link" when {
+  for(status: CaseStatus <- CaseStatus.values.filter((x => x == CaseStatus.COMPLETED  || x == CaseStatus.CANCELLED))) {
```

Comment:
why not just create a Seq(COMPLETED,CANCELLED) here.  You know what you want.

Reply:
True. Have improved it.